packages/plugins/sentry/src/index.ts

```diff
@@ -63,7 +63,11 @@ export type SentryPluginOptions = {
   /**
    * Indicates whether or not to skip the entire Sentry flow for given GraphQL operation
    */
-  skip?: (args: ExecutionArgs) => boolean;
+  skipOperation?: (args: ExecutionArgs) => boolean;
+  /**
+   * Indicates whether or not to skip the entire Sentry flow for given error
+   */
+  skipError?: (args: Error) => boolean;
```

Comment:
Technically the error should always be a `GraphQLError`. In which scenario would it "only" be an `Error`?

Reply:
Doesn't `GraphQLError` extend `Error`? I could be mistaken. This PR is kind of a sketch to get something in play.

Reply:
Happy to take your lead on this as I don't use TypeScript

Reply:
Let's keep it `Error` for now, I am not 100% sure, we can still refine it later.